ulog/log.go

```diff
@@ -51,6 +53,9 @@ type Log interface {
 	// RawLogger returns underlying logger implementation (zap.Logger) to get around the ulog.Log interface
 	RawLogger() zap.Logger
 
+	// Sentry returns the underlying sentry hook
+	Sentry() *sentry.Hook
```

Comment:
I remove this method, problem I'm running into is, how do we test this in service_test. Added a TODO for now. Let me know what you think.